tests/aediis-1/input.dat

```diff
@@ -0,0 +1,23 @@
+# ADIIS test case, from 10.1063/1.3304922 
+
+molecule {
+2 1
+Cd       0.0000000000      0.0000000000      0.0000000000    
+N        0.0000000000      0.0000000000     -2.2600010635    
+N       -0.6854437203      0.0000000000     -4.3480354859    
+C        0.6760533303      0.0000000000     -4.3850694419    
+C        1.0852404753      0.0000000000     -3.0912317094    
+C       -1.0447523879      0.0000000000     -3.0602202054    
+H        1.2315299626      0.0000000000     -5.3007589304    
+H        2.0886410569      0.0000000000     -2.7110772355    
+H       -2.0687499916      0.0000000000     -2.7265149874    
+H       -1.3131700977      0.0000000000     -5.1747180497    
+}
+
+set guess core    # With this guess, un-accelerated DIIS will fail.
+set scf_type pk
+set basis 3-21G
+set initial_scf_accelerator adiis
```

Comment:
Maybe reduce the number of maximum SCF iterations a little? This way one can be sure that convergence with ADIIS remains "consistently good".

Reply:
How tight do we want the number to be? Do we want to be alerted if the number increases _at all_, or just if it increases significantly?

Whatever we apply here, I'll do to `aediis-2` as well.

Reply:
I'd say fix it to the number of iterations it converges in on your machine + 1.